CONTRIBUTING.md

```diff
@@ -1,55 +1,160 @@
-# Contributing to knife-ec2
+# Contributing to Knife-EC2
```

Comment:
your shift key is errantly firing a lot on this line. might wanna look into that.